news/_posts/core-weekly/2022-08-08-coreweekly-31-2022.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 31 of 2022"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2022-08-08
+authors: [ PrestaShop ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+twitter_image: /assets/images/theme/banner-core-weekly.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 1st to Sunday 7th of August 2022.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+[write a nice message here, or remove the "General messages" section]
+
+
+## A quick update about PrestaShop's GitHub issues and pull requests:
+
+- [53 new issues](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+created%3A2022-08-01..2022-08-07) have been created in the project repositories;
+- [38 issues have been closed](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+closed%3A2022-08-01..2022-08-07), including [12 fixed issues](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Aissue+label%3Afixed+closed%3A2022-08-01..2022-08-07) on the core;
+- [45 pull requests have been opened](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+created%3A2022-08-01..2022-08-07) in the project repositories;
+- [41 pull requests have been closed](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+closed%3A2022-08-01..2022-08-07), including [35 merged pull requests](https://github.com/search?q=org%3APrestaShop+is%3Apublic++-repo%3Aprestashop%2Fprestashop.github.io++is%3Apr+merged%3A2022-08-01..2022-08-07).
+        
+
+
+## Code changes in the 'develop' branch
+
+
+### Back office
+* [#28820](https://github.com/PrestaShop/PrestaShop/pull/28820): Add employee info on the Payments tab when payment is registered, by [@Progi1984](https://github.com/Progi1984)
+* [#28799](https://github.com/PrestaShop/PrestaShop/pull/28799): Sort product table (in email, invoices and order view) by reference id, by [@Progi1984](https://github.com/Progi1984)
+
+
+## Code changes in the '8.0.x' branch
+
+
+### Core
+* [#29261](https://github.com/PrestaShop/PrestaShop/pull/29261): Bump distribution API module to 1.0.1, by [@matks](https://github.com/matks)
+* [#29239](https://github.com/PrestaShop/PrestaShop/pull/29239): Update composer deps and lock classic to beta-2.0.0, by [@matks](https://github.com/matks)
```

Comment:
Maybe interesting to highlight: we have created a tag on classic-theme to sync Core and Classic composer deps